tests/test_add_start_node.py

```diff
@@ -0,0 +1,38 @@
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver import ActionChains
+from selenium.common.exceptions import NoSuchElementException
+
+web = "https://machinist.flapflap.io/"
+
+
+def test_add_node(driver: WebDriver):
+
+    driver.get(web)
+    source = driver.find_element(By.XPATH, '//*[@id="__next"]/main/div[1]')
+    action = ActionChains(driver)
+    action.double_click(source).perform()
```

Comment:
I think you can also double click at a specific then double click however I am  not sure how to do it right now  and need to investigate further but if anyone else has the answer let me know because it could be helpful for one of my tests as well